src/tree/tree.py

```diff
@@ -0,0 +1,21 @@
+import argparse , sys , os
+
+def visit(path, depth, max_depth):
+    files = os.listdir(path)
```

Comment:
add try-except blocks to handle PermissionError and FileNotFoundError exceptions that may occur during directory listing.